### DISCUSSION TITLE
How to find deleted sessions?

### DISCUSSION THREAD

**Discussion by 00-kat** [sienori/Tab-Session-Manager#1248] [Q&A]
A session of mine suddenly disappeared, and I suspect it got deleted last night. I would like to get that session back somehow. I already checked the session-store folder in Firefox, which does not have it, and I am not sure how I can get it from Google Drive either (I think I haven't backed up since then). Would it be possible to find the deleted session?
Or does the history feature work even if you restart the browser?

**Comment by rohoso** [comment]
I found my backup folder in my Downloads folder.
\Downloads\TabSessionManager - Backup
But yeah, going forward I'm syncing it with my Google drive.
